src/stores/data.ts

```diff
@@ -270,5 +285,20 @@ export const useDataStore = defineStore('dataStore', {
       }
     },
 
+    // ------------------------------------- TOOLS ------------------------------------
```

Comment:
possible to create utilitie file for function like this, if its used in different files